docs/source/v2.1/WaterCycle/simulation_data/index.rst

```diff
@@ -0,0 +1,56 @@
+***************
+Simulation Data 
+***************
+
+The E3SMv2.1 simulation data is available on **ESGF** and **NERSC HPSS**.
+
+The preferred retrieval method is **ESGF**. Native output is available at `ESGF <https://esgf-node.cels.anl.gov/search?project=CMIP6&activeFacets=%7B%22source_id%22%3A%22E3SM-2-1%22%7D>`_, and a subset of the data is reformatted to conform to CMIP conventions and submmited to the CMIP6 ESGF archive (ESGF links are provided in the table below for published data).
```

Comment:
@forsyth2 
Sorry that I didn't catch this problem, but found it when reviewing the newsletter article. We did not publish v2.1 native data to ESGF. Could you update this to:
`
The data is reformatted to conform to CMIP conventions and submmited to the CMIP6 ESGF archive (ESGF links are provided in the table below for published data).`